src/components/Navbar.js

```diff
@@ -80,7 +80,7 @@ function Navbar() {
                     className='btn-link'
                     onClick={closeMobileMenu}
                   >
-                    <Button buttonStyle='btn--outline' buttonSize='btn--mobile'>
```

Comment:
The btn--mobile has a font-size of 24px which is huge for what we want it to be.